### PR TITLE
Trying out using the repo name from the action event

### PR DESCRIPTION
**Pull request by n3wscott**

### CODE REVIEW

File: hack/update-deps.sh [hunk 1]
```diff
@@ -61,8 +61,3 @@ rm -rf $(find vendor/ -name 'OWNERS')
 rm -rf $(find vendor/ -name '*_test.go')
 
 update_licenses third_party/VENDOR-LICENSE "./..."
-
-
-### HACK HACK HACK
-## Until we solve https://github.com/knative/pkg/issues/1590, let's do this:
-git apply ${ROOT_DIR}/hack/az-control-plane.patch
```
Comment (mattmoor) [inline hack/update-deps.sh:68]: why are you dropping this?

Comment (n3wscott) [reply hack/update-deps.sh:68]: https://github.com/knative/pkg/issues/1590 is solved

**Pull request merged**